apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -175,6 +177,11 @@ export class AddJob {
       throw new Error('Defer duration limit exceeded');
     }
 
+    await this.stepRunRepository.create(command.job, {
+      status: JobStatusEnum.DELAYED,
+      deferredMs: delay,
+    });
```

Comment:
Need to wrap this with if delay is actually hapening

Reply:
done.
now we set it as delayed for deferred job, and queued for others